sdk/src/wallet/update.rs

```diff
@@ -22,13 +22,9 @@ use crate::{
     wallet::events::types::{NewOutputEvent, SpentOutputEvent, TransactionInclusionEvent, WalletEvent},
 };
 
-impl<S: 'static + SecretManage> Wallet<S>
-where
-    crate::wallet::Error: From<S::Error>,
-    crate::client::Error: From<S::Error>,
```

Comment:
Can/should it be removed in more places?

Reply:
Ah I see you are removing in some places, what's the logic of removing or not?

Reply:
just isn't required

Reply:
it's needed whenever the secret manager is used basically